thirdparty/runtime_environments.go

```diff
@@ -308,3 +323,109 @@ func (c *RuntimeEnvironmentsClient) getHistory(ctx context.Context, opts DistroH
 	}
 	return amiHistory, nil
 }
+
+// stringToTime converts a string representing time to type time.Time.
+func stringToTime(timeInitial string) (time.Time, error) {
+	timestamp, err := strconv.ParseInt(timeInitial, 10, 64)
+	if err != nil {
+		return time.Time{}, errors.Wrapf(err, "converting time: '%s'", timeInitial)
+	}
+	return time.Unix(timestamp, 0), nil
+}
+
+// ImageInfo stores information about an image including its name, version_id, kernel, ami, and its last deployed time.
+type ImageInfo struct {
+	Name         string
+	VersionID    string
+	Kernel       string
+	LastDeployed time.Time
+	AMI          string
+}
+
+// GetDistroInfo returns information about a distro.
+func (c *RuntimeEnvironmentsClient) GetDistroInfo(ctx context.Context, imageID string) (*ImageInfo, error) {
```

Comment:
helper functions would probably be really useful here to make the code easier to read!